### PR TITLE
Add version number for group_by_exp doc

### PR DESCRIPTION
**Pull request by karel-3d**

### CODE REVIEW

File: docs/_docs/templates.md [hunk 1]
```diff
@@ -178,7 +178,7 @@ you come up with your own tags via plugins.
     <tr>
       <td>
         <p class="name"><strong>Group By Expression</strong></p>
-        <p>Group an array's items using a Liquid expression.</p>
+        <p>Group an array's items using a Liquid expression. Jekyll v3.4.0 & later.</p>
```
Comment (DirtyF) [inline docs/_docs/templates.md]: @karel-3d we have an include for that:
`{% include docs_version_badge.html version="3.4.0" %}`

Comment (karel-3d) [reply docs/_docs/templates.md]: I copy-pasted this from few lines earlier, where_exp. Should I change both?

Comment (DirtyF) [reply docs/_docs/templates.md]: yes, please.

File: docs/_docs/templates.md [hunk 2]
```diff
@@ -178,7 +179,8 @@ you come up with your own tags via plugins.
     <tr>
       <td>
         <p class="name"><strong>Group By Expression</strong></p>
-        <p>Group an array's items using a Liquid expression.</p>
+        <p>Group an array's items using a Liquid expression.
+        {% include docs_version_badge.html version="3.4.0" %}</p>
```
Comment (pathawks) [inline docs/_docs/templates.md]: Note: We don't want to use `{%- -%}` syntax here, because we want a space after the period before the badge.

**Comment by DirtyF** [comment]
@jekyllbot: merge +docs

**Comment by karel-3d** [comment]
Thanks!

**Pull request merged**